app-backend/ingest/build.sbt

```diff
@@ -0,0 +1,17 @@
+name := "rf-ingest"
+
+assemblyJarName in assembly := "rf-ingest.jar"
+
+mainClass in (Compile, assembly) := Some("com.azavea.rf.ingest.Ingest")
+
+javaOptions += "-Xmx8G"
```

Comment:
The really small tiles I cut and included can be handled with quite a bit less. Going to try pushing this down to 2G to see what happens.